### PR TITLE
Player hp

### PR DESCRIPTION
**Pull request by jsoconnell**

### CODE REVIEW

File: invadedhouse/models/room.py [hunk 1]
```diff
@@ -23,7 +25,7 @@ def enter(self, player):
         self.look(player)
 
     def look(self, player):
-        print(self.get_name() + ':')
+        cprint(self.get_name() + ':', 'cyan')
```
Comment (redjoey) [inline invadedhouse/models/room.py:28]: umm whats `cprint`?

File: README.md [hunk 2]
```diff
@@ -8,6 +8,15 @@
...
 
+## Install dependencies
+
+- From the command prompt, enter:
+    ```bash
+    python3 -m venv .venv
+    source .venv/bin/activate
+    pip install -r requirements.txt
+    ```
+
```
Comment (redjoey) [inline README.md:19]: what. is. that.

File: invadedhouse/models/room.py (hunk 1 above)
Comment (redjoey) [reply invadedhouse/models/room.py:28]: and whats termcolor

Comment (jsoconnell) [reply invadedhouse/models/room.py:28]: it's part of a library that allows us to print IN COLOR

**Review by redjoey — APPROVED** [review]
umm so we have player hp and we have this termcolor thing... What is that thing you added. Whats "termcolor"? Is it colors? I dont know but umm. Yeah. Player HP. I approve.

File: invadedhouse/models/room.py (hunk 1 above)
Comment (redjoey) [reply invadedhouse/models/room.py:28]: WOAH!

Comment (redjoey) [reply invadedhouse/models/room.py:28]: AWESOME

Comment (jsoconnell) [reply invadedhouse/models/room.py:28]: termcolor is the library, cprint is a function in the library. you'll see a nice surprise when you implement the "monsters attack back" feature...

File: README.md (hunk 2 above)
Comment (jsoconnell) [reply README.md:19]: these are three commands you have to run to get termcolor installed into the project

File: invadedhouse/models/room.py (hunk 1 above)
Comment (redjoey) [reply invadedhouse/models/room.py:28]: btw IDLE always prints in blue. But probably anything you do with this printing in color thing wont be blue in IDLE. But we have IDE!

Comment (redjoey) [reply invadedhouse/models/room.py:28]: also could we put optional instructions in the readme that will show people how to get IDE

Comment (redjoey) [reply invadedhouse/models/room.py:28]: but PRINTING IN COLOR WOAH

Comment (redjoey) [reply invadedhouse/models/room.py:28]: also what do i need to do to use this color thing because i need to install it i think right?

**Pull request merged**